test/relations.test.js

```diff
@@ -4877,6 +4877,47 @@ describe('relations', function() {
 
   });
 
+  describe('embedsMany - async validation', function() {
+
+    var Book, Page;
+
+    before(function(done) {
+      tmp = getTransientDataSource({defaultIdType: Number});
+      // db = getSchema();
+      Book = db.define('Book', {name: String});
+      Page = tmp.define('Page', {number: Number});
+      Page.validateAsync('number', function(err, done) {
+        var self = this;
+        process.nextTick(function() {
+          if (self.number < 0) err();
+          done();
+        });
+      });
+
+      db.automigrate(['Book'], done);
+    });
+
+    it('creation', function(done) {
+      Book.embedsMany(Page);
+      db.automigrate(['Book'], done);
+    });
```

Comment:
AFAICT you are not testing any functionality in:
```
    it('creation', function(done) {
      ...
    });
```

Shouldn't this be `before instead`?:
```
before(function(done) {
  ...
})
```